types/office-js/index.d.ts

```diff
@@ -1157,6 +1157,13 @@ declare namespace Office {
          * This is best for user experience and performance.
          */
         displayInIframe?: boolean
+        /**
+         * Determines if the pop-up blocker dialog will be shown to the user. Defaults to true.
+         * 
+         * `true` - The framework displays a pop-up to trigger the navigation and avoid the browser's pop-up blocker.
+         * `false` - The dialog will not be shown and the developer will handle pop-ups (usually by providing a user artifact to trigger the navigation).
```

Comment:
and the developer must handle pop-ups (by providing a user interface artifact to trigger the navigation)

Reply:
Fixed. Thanks!